### PR TITLE
Preparation for Release 29

### PR DESCRIPTION
**Pull request by StykMartin**
This pull request features various changes we need to move development forward.
This includes
- New CI pipeline for Integration Tests
- Various test fixes
- Reintroducing RHEL 7 build pipeline
- Docs fixes
- Build fixes
- Release 29 schema lock

### CODE REVIEW

File: .github/workflows/review-checks.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
     - uses: actions/checkout@v1
     - name: Install utils
       run: |
-        yum install -y git wget
+        yum install -y git wget ca-certificates -y
```
Comment (mdujava) [inline .github/workflows/review-checks.yml:30]: :eyes: `-y [...] -y` yum now knows you mean business

Comment (StykMartin) [reply .github/workflows/review-checks.yml:30]: 🤦🏻 Thanks

**Comment by StykMartin** [comment]
What is currently missing:
- Define alembic revision for 29
- Drop [cgi](https://docs.python.org/3/library/cgi.html)

Anything else on your mind @mdujava?

**Review by mdujava — APPROVED** [review]

**Comment by StykMartin** [comment]
One more to go
```
ERROR: test_cannot_import_osmajor_existing_alias (bkr.inttest.labcontroller.test_distro_import.DistroImportTest)
```

**Comment by StykMartin** [comment]
@mdujava @p3ck PTAL

**Comment by StykMartin** [comment]
I will deliver CI/CD to deploy release in additional PR, the scope here is big enough already.

**Review by mdujava — APPROVED** [review]

**Review by p3ck — APPROVED** [review]
LGTM!

**Pull request merged**